### PR TITLE
Awesome book

### PR DESCRIPTION
**Pull request by Kandy-Peter**
This pull request contains the following steps and elements:

- use ES6 syntax to create the awesome book
- create a module folder containing all the modules files
- get access to modules from index.js
- use packages management
- use luxon npm package to get data
- fix linters and bugs

### CODE REVIEW

File: modules/book.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+  addNewBook(book) {
+    this.bookArray.push(book);
+    localStorage.setItem('bookstore', JSON.stringify(this.bookArray));
+  }
+
+  removeBook(id) {
+    this.bookArray = this.bookArray.filter((book) => book.id !== Number(id));
+    localStorage.setItem('bookstore', JSON.stringify(this.bookArray));
+  }
```
Comment (mikethreels) [inline modules/book.js:14]: - [ ] according to the requirements you should use ES6 syntax for this milestone, which means that all functions you created should be arrow functions

File: modules/displayBook.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  newAddDiv.classList.add('bookItem');
+  newAddDiv.id = book.id;
+  newAddDiv.innerHTML = `<ul class="book-content">
+                                <li class = "title">${book.title}</li>
+                                <li>${book.author}</li>
+                            </ul>
+                            <button id="remove">Remove</button>
+    `;
+  booksContainer.appendChild(newAddDiv);
+}
```
Comment (mikethreels) [inline modules/displayBook.js]: - [x] according to the requirements you should use ES6 syntax for this milestone, which means that all functions you created should be arrow functions, which in this case might look something like below:
```
const displayBook = (book) => {
  const newAddDiv = document.createElement('div');
  const booksContainer = document.querySelector('.displayBook');
  newAddDiv.classList.add('bookItem');
  newAddDiv.id = book.id;
  newAddDiv.innerHTML = `<ul class="book-content">
                                <li class = "title">${book.title}</li>
                                <li>${book.author}</li>
                            </ul>
                            <button id="remove">Remove</button>
    `;
  booksContainer.appendChild(newAddDiv);
};

export default displayBook;
```

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Kandy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/book.js (hunk 1 above)
Comment (Kandy-Peter) [reply modules/book.js:14]: Hi @mikethreels,
the javascript class replaces the function in this case, so `addBook and `removeBook` are just methods inside the class and it's a correct syntax CFR [https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Classes](url)

**Review by julie-ify — APPROVED** [review]
## Hi @Kandy-Peter, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**